source/mir/ndslice/slice.d

```diff
@@ -997,6 +997,12 @@ public:
         return typeof(return)(_lengths[d].lightImmutable, _labels[d]);
     }
 
+    /// Strips label off the DataFrame
+    auto stripLabels()()
```

Comment:
should be just named `values` like in pandas,
Also, should be two more versions: for `const` and `immutable` struct.

Reply:
renamed to `values`. Added `valuesConst` and `valuesImmutable` in 4c97c4